### PR TITLE
Fixes #3340 - Run js unit tests in CI

### LINKED ISSUE #3340: Run javascript unit tests in CI
Modify https://github.com/webcompat/webcompat.com/blob/master/.circleci/config.yml and add `npm run test:unit-js`

### PR DESCRIPTION
**Pull request by ksy36**
r? @miketaylr

### CODE REVIEW

**Comment by ksy36** [comment]
Oops this is not ready for review yet, sorry.

**Comment by miketaylr** [comment]
> Oops this is not ready for review yet, sorry.

no worries

File: webcompat/static/js/lib/wizard/tests/helpers/babel-register.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+require("@babel/register")({
+  babelrc: false,
+  presets: ["@babel/env"],
+});
```
Comment (ksy36) [inline webcompat/static/js/lib/wizard/tests/helpers/babel-register.js:4]: Here babel register doesn't use default `.babelrc` and transpiles everything (similarly to webpack config for es5). This allows to use es6 modules in unit tests in node environment

**Comment by ksy36** [comment]
ready for review @miketaylr :)

**Pull request merged**